example/src/App.tsx

```diff
@@ -1,21 +1,20 @@
 import * as React from 'react';
```

Comment:
Is this just used for testing? Or would we actually render this?

Reply:
Yeah it's an example app file that's set up to run against the local package so you can make changes and test easily - it's not used when you install it elsewhere 👍